todo/script.js

```diff
@@ -0,0 +1,84 @@
+"use stric"
+/* вывод списка дел , второе зачеркиваение 
+статус задачи done: bool;
+id: string;
+
+чтобы вывести список нам нужно вывести элементы массива
+чтобы удалить находим id и изменяем с try на false
+*/
+
+// учим работать с контейнером todos
+const todosNode = document.querySelector('.js-todos');
+const inputNode = document.querySelector('.js-input');
+const btnNode = document.querySelector('.js-btn');
+let todos = [];
+
+function addTodo(text) {
+   const todo = {
+      text,
+      done: false,
+      id: `${Math.random()}`
+   };
+
+   todos.push(todo);
+} // добавление тудушки 
+
+// удаление тудушки 
+
+function deleteTodo(id) {
+   todos.forEach(todo => {
+      if (todo.id === id) {
+         todo.done = true; 
+      }
+   })
```

Comment:
done - это значит, что тудушка выполнена, а не удалена
тут нужно удалить из массива todos  тудушку
```suggestion
   todos = todos.filter(item => item.id !== id);
```